docs/language/accounts.mdx

```diff
@@ -14,10 +14,30 @@ Every account can be accessed through two types:
       let storageUsed: UInt64
       let storageCapacity: UInt64
 
+      // Keys
+      let keys: AuthAccount.Keys
+
       // Storage operations
 
       fun getCapability<T>(_ path: PublicPath): Capability<T>
       fun getLinkTarget(_ path: CapabilityPath): Path?
+
+      struct Keys {
```

Comment:
This struct is also defined in the `AuthAccount` struct below. Should it be in both places?

Reply:
Good point! `PublicAccount.Keys` only supports `get` method, whereas `PublicAccount.Keys` supports `add`, `get`, and `revoke`. Hence they had to be implemented as two different structs.

But I have accidentally added the other two methods also, in the documentation for the `PublicAccount.Keys`. Removed it in 98aaf74.
Thanks for catching that!